### DISCUSSION TITLE
👋 Who is using GraphQL CLI?

### DISCUSSION THREAD

**Discussion by kamilkisiela** [Urigo/graphql-cli#1163] [General]
We would love to hear who's using GraphQL CLI and how you're using it :)

If it's okay to share, please mention the name of your company or product so we could include it on a special section on the website.

**Comment by wtrocki** [comment]
We are using it extensively in AeroGear:

- GraphQL config (indirectly)
- Codegen (main use case)
- Generate (obviously)

Never used inspector tbh.
